src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -296,7 +296,7 @@ public static void UpdateMaximumWidth(this AView platformView, IView view)
 			PlatformInterop.RequestLayoutIfNeeded(platformView);
 		}
 
-		public static async Task UpdateBackgroundImageSourceAsync(this AView platformView, IViewBackgroundImagePart viewBackgroundImagePart, IImageSourceServiceProvider? provider)
+		public static async Task UpdateBackgroundImageSourceAsync(this AView platformView, IImageSource? imageSource, IImageSourceServiceProvider? provider)
```

Comment:
This is technically a breaking change, but I don't think it is used in RC3 by anyone and will be quickly replaced by RC4/GA.